### DISCUSSION TITLE
Bounty Master disappears

### DISCUSSION THREAD

**Discussion by swordfishmetal1** [Aelto/W3_RandomEncounters_Tweaks#85] [General]
The marker showed up earlier in game, but the Bounty Master disappeared after I completed the elven ruins with Keira

**Comment by Aelto** [comment]
Hello,

He moves around on the map as you progress in the game. Have you searched everywhere on the map for his marker?

**Comment by swordfishmetal1** [comment]
So I started as new game. with 2.9.2  (Vanilla) and there are no contracts on the job board and there's no marker for Graden. But the RER mod menu is enabled, all the options seemed to be loaded correctly. Is there a delay for the new RER to start?

**Reply by Aelto** [reply]
Hello!

This is a new feature, during fresh playthroughs the mod now slowly activates over the first ten hours of gameplay. It is not yet documented since I'm planning a [Getting started](https://aelto.github.io/tw3-random-encounters-reworked/rer-bible/tldr) page that will explain it.

- after 1 hour: encounters are now being created
- after 2 hours: the bounty master and the contracts in the boards are added
- after 3 hours: the events are activated

Then for the next seven hours the encounters are created at -70% of their intended frequency, plus 10% for every successive hour of gameplay . Same for the events.

This ensures a smooth start while you're still gathering stuff and the basic perks for your build. After these 10 hours the mod will work at its full potential.

This feature is not tied to the settings and you can't change it yourself.
